recipes/cytosim/LICENSE.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
This file is in the tarball. No need to add it twice. Please remove it from the PR.

Reply:
Actually it there is an issue when I run the docker build without a license file.
During the packaging of cytosim, I get : 

Fixing permissions
Traceback (most recent call last):
(...)
 raise ValueError(
ValueError: License file given in about/license_file (/home/conda/staged-recipes-copy/recipes/cytosim/LICENCE.txt) does not exist in source root dir or in recipe root dir (with meta.yaml)

##[error]Bash exited with code '1'.